src/components/common/base-card/base-card.styles.tsx

```diff
@@ -13,7 +13,7 @@ interface IBodyStyleProps {
 // =============================================================================
 export const Body = styled.div<IBodyStyleProps>`
     padding: 1rem;
-    border-radius: 0.5rem;
+    border-radius: ${Radius.sm};
```

Comment:
this needs to set after `Border.Util.solid` or passed as a param too, as the util overrides border-radius. will document this behaviour